### PR TITLE
feature(editors): quill presets

### PR DESCRIPTION
**Pull request by pweyrich**
@plentymarkets/team-terra

#### Must be done by Terra

Testing
- [x] Person 1 (mandatory)
- [ ] Person 2 (mandatory)

Information transfer
- [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Browser-Support (relevant for changes in scss / appearance of components and jQuery)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Terra Basic Plugin
- [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

----

#### Must be done by every developer
Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline). 

Documentation
- [x] Changelog (optional: relevant for every change which influences the API)
- [x] Example (updated or created)
- [ ] JSDoc (optional: relevant for every change which influences the API)

Testing
- [ ] Unit Test (updated or created)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 * **terraDraggable** deprecate directive.
 * **terraDropzone** deprecate directive.
 * **terraResizable** deprecate directive.
+* **quill-presets** introduced two presets (base and note) for [quill](https://quilljs.com/) rich text editor.
+* **terra-note-editor** is deprecated now. Use `<quill-editor>` component provided by [ngx-quill](https://github.com/KillerCodeMonkey/ngx-quill) and our note preset instead.
```
Comment (niwolf) [inline CHANGELOG.md:9]: terra-base-editor --> deprecated ... sollte auch dazu oder?

Comment (pweyrich) [reply CHANGELOG.md:9]: Der ist nicht Teil der public-api. Wollte ihn nur so markieren, damit wir wissen, dass wir ihn bald entfernen können.

**Comment by niwolf** [comment]
Changelog muss noch geupdated werden

**Comment by pweyrich** [comment]
> Changelog muss noch geupdated werden

Stimmt. Danke!

**Comment by pweyrich** [comment]
stale

**Pull request closed without merging**